crates/ruff_linter/src/rules/airflow/rules/moved_to_provider_in_3.rs

```diff
@@ -59,6 +62,7 @@ impl Violation for Airflow3MovedToProvider {
     fn fix_title(&self) -> Option<String> {
         let Airflow3MovedToProvider { replacement, .. } = self;
         match replacement {
+            ProviderReplacement::None => {None}
```

Comment:
```suggestion
            ProviderReplacement::None => None,
```

I'm surprised clippy or rustfmt didn't catch this.

Reply:
updated. thanks!